caddy/Caddyfile.pgs

```diff
@@ -38,8 +69,10 @@
 			regex {
 				exclude /check
 			}
+			key {
+				template {http.request.method}-{http.request.path}
+			}
 		}
-		reverse_proxy web:3000
```

Comment:
@mac-chaffee did we need this `rever_proxy` directive if it is in the parent address already?

Reply:
Right I think we don't need both